streams/src/main/java/org/apache/kafka/streams/processor/internals/CompositeRestoreListener.java

```diff
@@ -1,116 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This class is actually not used anywhere except unit tests: it is only a by-product of the coupling.

Reply:
_awesome_